src/js/requestsFactory.js

```diff
@@ -1,21 +1,32 @@
 const errorHandler = (async() => await import('./errorHandler.js'))();
```

Comment:
actually, lazy loading doesn't work,
change the line to 
`const errorHandler = (async() => await import(/* webpackChunkName: "errorHandler" */'./errorHandler.js'))();`

and see to the network tab in the browser.

Reply:
get rid of all the IIFE they are not needed